README.md

```diff
@@ -1 +1,57 @@
-"# raster-sdk" 
+![Precisely](Precisely_Logo.png "Precisely")
+# RasterSDK Sample
+This sample gradle project of RasterSDK demonstrates the usage of various RasterSDK APIs.
+The `/src/` folder contains sample code of different APIs and their usage.
```

Comment:
If we can provide a hyperlink to the actual documentation/user guide I believe it will be helpful for teams accessing the APIs

Reply:
As user guide's location TBD, it think its better to create a jira in backlog to update this readme.me file once we have user guide on web.